### PR TITLE
[3510] add rollover env

### PR DESCRIPTION
**Pull request by misaka**
### Context

We're building out a rollover env to test rollover.

### Changes proposed in this pull request

Add a `rollover` Rails env.

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [ ] Tested by running locally
- [ ] Product Review

### CODE REVIEW

**Comment by misaka** [comment]
[Add rollover RAILS_ENV](https://trello.com/c/bi6frhLQ/3510-add-rollover-railsenv)

File: config/settings/rollover.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+dfe_signin:
+  secret: please_change_me # Override with SETTINGS__DFE_SIGNIN__SECRET
+  base_url: https://s121d04-rollover-publish-as.azurewebsites.net
+manage_backend:
+  base_url: https://s121d04-rollover-ttapi-as.azurewebsites.net
+environment:
+  label: "QA"
+  selector_name: "qa"
```
Comment (defong) [inline config/settings/rollover.yml:8]: are you sure `QA`?

Comment (misaka) [reply config/settings/rollover.yml:8]: It's the same as what we did for `research`, so I _think_ that's correct. I'll double-check.

Comment (misaka) [reply config/settings/rollover.yml:8]: Let's give it a go and see what happens, it'd be nice if it worked.

**Pull request merged**